src/main/flight/pid.c

```diff
@@ -212,10 +212,9 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .idle_p = 50,
         .idle_pid_limit = 200,
         .idle_max_increase = 150,
-        .ff_interpolate_sp = 0,
-        .ff_spread = 0,
+        .ff_interpolate_sp = FF_INTERPOLATE_AVG,
+        .ff_jerk_limit = 250,
```

Comment:
Was it intentional to set the jerk limit to 250? 
In my testing, 25 was the optimal value.
250 would effectively disable the limiting of spikes in the boost signal.  If the intent was to have no limiting, 250 is the right value.  But if the intent is to make it work best, it should be 25.  The latter seems better.